.changeset/gentle-dolphins-draw.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Empty changeset file?

Reply:
Yep, since it's not released, it's unhelpful to write incremental changes in the public changelog (because from the user's perspective, nothing changed). `npx changeset --empty` creates an empty one which is what the failing changeset action suggests to do to silence it.